src/main/java/com/contentful/java/cma/ModuleUploads.java

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright (C) 2017 Contentful GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.contentful.java.cma;
+
+import com.contentful.java.cma.RxExtensions.DefFunc;
+import com.contentful.java.cma.model.CMAUpload;
+import com.contentful.java.cma.model.CMAUploadWelcome;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Executor;
+
+import okhttp3.MediaType;
+import okhttp3.RequestBody;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+
+import static java.lang.System.arraycopy;
+import static java.util.Arrays.copyOf;
+
+/**
+ * Upload Module.
+ * <p>
+ * This module will take care of all `upload.contentful.com` related issues, as in directly
+ * uploading a file to contentful, receiving it's id.
+ */
+public final class ModuleUploads extends AbsModule<ServiceUploads> {
+  private static final String CONTENT_TYPE_STREAM = "application/octet-stream";
```

Comment:
Here it's called CONTENT_TYPE_STREAM, what's the difference between this and OCTET_STREAM_CONTENT_TYPE?